bin/varnishtest/tests/c00136.vtc

```diff
@@ -0,0 +1,20 @@
+varnishtest "none probe"
+
+server s0 {
+	rxreq
+	txresp -status 500
+} -dispatch
+
+varnish v1 -vcl {
+	probe default {
+		.interval = 100ms;
+	}
+	backend s0 {
+		.host = "${s0_sock}";
+		.probe = none;
```

Comment:
Can we also make sure please that this also works for a `VCL_PROBE` vmod function/method argument?

Reply:
I don't think we can pass `none` for a backends either, that would be a subsequent change (among a couple others) should this one be approved.

Reply:
We can always define a none backend and then pass that, so that is not so important. For probes we don't have that option.

Reply:
Right, so you'd like to align this change with backend definitions?

```vcl
probe disabled none;

backend improbable {
       .probe = none;
}

backend also_improbable {
       .probe = disabled;
}
```

That would increase overall consistency, good idea.